source/features/jump-to-conversation-close-event.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function init(): Deinit {
+	return observe('#partial-discussion-header :is([title="Status: Closed"], [title="Status: Merged"], [title="Status: Closed as not planned"]):not(.rgh-jump-to-conversation-close-event)', {
+		constructor: HTMLSpanElement,
+		add(discussionHeader) {
+			discussionHeader.classList.add('rgh-jump-to-conversation-close-event');
+			// Hide the native title
+			discussionHeader.style.pointerEvents = 'none';
+			discussionHeader.style.cursor = 'default';
+
+			const lastCloseEvent = select.last('.TimelineItem-badge :is(.octicon-issue-closed, .octicon-git-merge, .octicon-git-pull-request-closed, .octicon-skip)')!.closest('.TimelineItem')!;
```

Comment:
:is/:has surely makes selectors unreadable 😰 

Can these be replaced by `color-bg-closed-emphasis` and the purple equivalent?

Also if you'd like to extend our new `selectHas` helper with `selectHas.last` it'd be great. I think it _just_ needs an `if (last) array.reverse()` somewhere in it

Reply:
> Can these be replaced by color-bg-closed-emphasis and the purple equivalent?

Would be the same amount of selectors (They each have their own) and I think it would be more fragile.